wbooks/src/app/screens/Library/style.ts

```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+import { aliceBlue } from '@constants/colors';
+
+const style = StyleSheet.create({
+  libraryContainer: {
+    display: 'flex',
+    paddingVertical: 15,
+    alignItems: 'center',
+    paddingHorizontal: 15,
```

Comment:
paddingVertical and paddingHorizontal share the same value, extract into a constant